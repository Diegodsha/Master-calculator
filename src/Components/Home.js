import React from 'react';
import styled from 'styled-components';
import { TextWithShadow } from './Calculator';
import welcome from '../assets/welcome.jpg';

const StyledHome = styled.div`
  background-color: #ffffff82;
  border-radius: 0.5rem;
  img {
    width: 500px;
    border-radius: 0.5rem;
    margin: 16px 0;
  }
`;

const Home = () => (
  <StyledHome className="container text-center">
    <TextWithShadow className="text-white display-2">
      Welcome to Math Magicians!
    </TextWithShadow>
    <img className="img-fluid" src={welcome} alt="welcome" />
    <TextWithShadow className="text-white">
      We are an enthusiastic team based in France that loves teaching and
      helping with all topics related to Mathematics. Our goal is that you start
      loving it as much as we do by providing efficient, fun to use and,
      beautifully designed web and mobile tools (free to use of course) that
      will help you through your learning path and will enhance your experience
      as a Mathematics lover.
    </TextWithShadow>
  </StyledHome>
);

export default Home;
