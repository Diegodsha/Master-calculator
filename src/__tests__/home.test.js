import renderer from 'react-test-renderer';
import Home from '../Components/Home';

describe('Home Component', () => {
  test('Renders correctly home component in the Dom', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
