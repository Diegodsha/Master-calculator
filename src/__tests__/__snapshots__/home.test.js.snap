// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Renders correctly home component 1`] = `
<div
  className="sc-eCApnc moOtr container text-center"
>
  <p
    className="sc-hKFxyN LYQFs text-white display-2"
  >
    Welcome to Math Magicians!
  </p>
  <img
    alt="welcome"
    className="img-fluid"
    src="welcome.jpg"
  />
  <p
    className="sc-hKFxyN LYQFs text-white"
  >
    We are an enthusiastic team based in France that loves teaching and helping with all topics related to Mathematics. Our goal is that you start loving it as much as we do by providing efficient, fun to use and, beautifully designed web and mobile tools (free to use of course) that will help you through your learning path and will enhance your experience as a Mathematics lover.
  </p>
</div>
`;
